rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -3827,6 +3829,8 @@ static void rmw_fini_cs(CddsCS * cs)
   dds_delete(cs->sub->rdcondh);
   dds_delete(cs->sub->enth);
   dds_delete(cs->pub->enth);
+  delete cs->sub;
+  delete cs->pub;
```

Comment:
store NULL after `delete` to avoid double free?

Reply:
Actually, rmw_cyclonedds seems not to provide extra source code to avoid double free, I followed the same style as following,

https://github.com/ros2/rmw_cyclonedds/blob/4fc5de7aa55d53d3cdd72aa9ae7d890ccdfefbe0/rmw_cyclonedds_cpp/src/rmw_node.cpp#L1349-L1350

Reply:
Anyway, thanks for your suggestion, I'll update it later.

Reply:
> I followed the same style as following,

i see that, i understand. but it would be better for code quality i guess.